packages/ai-sdk-provider-grok-cli/README.md

```
@@ -0,0 +1,167 @@
+## Installation
+
+```bash
+npm install @tm/ai-sdk-provider-grok-cli
+# or
+yarn add @tm/ai-sdk-provider-grok-cli
+```
+
+## Prerequisites
+
+1. Install the Grok CLI:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)